physics/module_bl_mynn.F90

```diff
@@ -3794,29 +3800,31 @@ SUBROUTINE  mym_condensation (kts,kte,   &
 
         !Diagnostic statistical scheme of Chaboureau and Bechtold (2002), JAS
         !but with use of higher-order moments to estimate sigma
-        PBLH2=MAX(10.,PBLH1)
+        pblh2=MAX(10.,pblh1)
```

Comment:
To avoid precision loss when calling intrinsics you should be explicit about the precision (e.g 10._kind_phys instead of 10.)

Reply:
fixed